src/app/content/highlights/components/Note.tsx

```diff
@@ -47,6 +48,12 @@ const SimpleLabel = styled.label`
   padding-left: 0.2rem;
 `;
 
+// exported for test coverage reasons
+export function escapeHandler(onElement: HTMLTextAreaElement | null, shouldDo: boolean) {
+  if (shouldDo) {
```

Comment:
it's unclear to me why this boolean is passed into the function, the check could just live outside, in escCb

Reply:
The boolean was the reason I needed to separate/export the function to get test coverage. The condition is always false in testing due to how refs are handled.